<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display Locker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4">
     <widget class="QTableWidget" name="tableWidgetSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>长</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>宽</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>颜色位数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>刷新率</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonApply">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonLock">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>锁定</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelInfoNow">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>149</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionAutoRun"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionAutoRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>开机后台运行</string>
   </property>
  </action>
  <action name="actionShowtray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>最小化时隐藏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonApply</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidgetSettings</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsCellActivated(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLock</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>lock(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>settingsCellActivated(int,int)</slot>
  <slot>lock(bool)</slot>
 </slots>
</ui>
